src/common/metrics/case_growth.ts

```diff
@@ -36,6 +39,13 @@ export const CASE_GROWTH_RATE_LEVEL_INFO_MAP: LevelInfoMap = {
     color: COLOR_MAP.ORANGE.BASE,
     detail: () => SHORT_DESCRIPTION_MEDIUM,
   },
+  [Level.MEDIUM_HIGH]: {
+    level: Level.MEDIUM_HIGH,
+    upperLimit: LIMIT_MEDIUM_HIGH,
+    name: MEDIUM_HIGH_NAME,
+    color: '#ff6d00',
```

Comment:
Huh.  I wonder if there's a good reason we don't use LEVEL_COLOR constants for these...  🤔  Seems like something we should maybe clean up at some point (not in this PR).

Reply:
I'm going to add that color to `COLOR_MAP` for now as `ORANGE_DARK` so the color is not hardcoded here, but yes, I think that ideally, we should use `LEVEL_COLOR`. I can fix that on a separate PR but want to check with @goldblatt first

Reply:
I think it's because the colors are reversed for contact tracing https://github.com/covid-projections/covid-projections/pull/850/files#diff-b88c1dc9a0f512264e2d89dc4032053a